### PR TITLE
feat(acme) add rsa_key_size config option to acme plugin

### PR DESCRIPTION
**Pull request by lodrantl**
### Summary

Add rsa_key_size configuration option to the acme plugin as mentioned in https://github.com/Kong/kong/discussions/8111.

### Full changelog

* adds the configuraiton option
* no tests were added  (cert_type option which is similar has none either)

### CODE REVIEW

File: kong/plugins/acme/schema.lua [hunk 1]
```diff
@@ -2,6 +2,8 @@ local typedefs = require "kong.db.schema.typedefs"
 
 local CERT_TYPES = { "rsa", "ecc" }
 
+local RSA_KEY_SIZES = {2048, 3072, 4096}
```
Comment (fffonion) [inline kong/plugins/acme/schema.lua:5]:
```suggestion
local RSA_KEY_SIZES = { 2048, 3072, 4096 }
```

File: kong/plugins/acme/schema.lua [hunk 2]
```diff
@@ -83,6 +85,11 @@ local schema = {
           default = 'rsa',
           one_of = CERT_TYPES,
         }, },
+        { rsa_key_length = {
+          type = "number",
+          default = 4096,
+          one_of = RSA_KEY_LENGTHS,
```
Comment (fffonion) [inline kong/plugins/acme/schema.lua:91]:
```suggestion
          one_of = RSA_KEY_SIZES,
```

**Review by lodrantl — COMMENTED** [review]
Forgot to commit the changes to schema.lua after renaming the rsa_key_length to rsa_key_size which I think makes more sense. Fixing the mistake here

**Comment by fffonion** [comment]
Looks good to me, @lodrantl since we are in feature freeze, will merge it in 2.8.x.

**Review by fffonion — APPROVED** [review]

**Comment by fffonion** [comment]
Just merged, @lodrantl thanks again for your PR!

**Pull request merged**